src/solvers/flattening/boolbv.h

```diff
@@ -271,6 +271,9 @@ class boolbvt:public arrayst
 
   // strings
   numberingt<irep_idt> string_numbering;
+
+  // scopes
+  std::size_t scope_counter = 0;
```

Comment:
Random place for comment:
- In the commit message fix s/looses/loses/
- The commit message promises "(which is the next  commit)" -- but there is no such change to be found.

Reply:
Commit message fixed.